.github/ISSUE_TEMPLATE/04-github-request---repo-settings.yml

```diff
@@ -0,0 +1,53 @@
+name: ⚙️ GitHub Request - Repository Settings
+description: Make non-permissions settings changes to a repository in the openedx organization
+title: "[GH Request] <ADD A REQUEST TITLE HERE>"
+labels: ["github-request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## GitHub Request - Repository Settings
+        Please leave [GH Request] in the title of your ticket!
+        
+        Use this ticket to request specific settings changes or ask questions about
+        repository settings. Note that you can view all of the available settings 
+        in your own repositories or forks in your personal account.
+        
+        If you are not the maintainer on the repository we may add them to the 
+        ticket to make sure the permissions make sense to the maintainers.
+  - type: input
+    id: repository
+    attributes:
+      label: Repository
+      description: Name of the repository you would like altered
+      placeholder: ""
```

Comment:
I think this line may be causing the error I left a comment about, do we need to set a placeholder value here? Would it work if we just removed this line?

```suggestion
```

Reply:
I think you just omit the `""`: https://github.com/openedx/axim-engineering/blob/main/.github/ISSUE_TEMPLATE/01-github-request---onboarding.yml?plain=1#L16

Reply:
It looks like `placeholder:` (with nothing after it) might do the trick?

https://github.com/openedx/axim-engineering/blob/bf23b6371b719d374e04dc04f52cb9dc4c4c7356/.github/ISSUE_TEMPLATE/03-github-request---access.yml?plain=1#L19

Reply:
Bah, of course it's the line my editor told was an error and insisted on fixing.